nextflow_schema.json

```
@@ -726,9 +726,204 @@
+                    "fa_icon": "fas fa-align-left"
+                },
+                "metagenomics_maltextract_ncbidir": {
+                    "type": "string",
+                    "description": "Path to directory containing containing NCBI resource files (ncbi.tre and ncbi.map; available: https://github.com/rhuebler/HOPS/)",
+                    "default": null,
+                    "help_text": "Path to directory containing containing the NCBI resource tree and taxonomy table files (ncbi.tre and ncbi.map; available at the [HOPS repository](https://github.com/rhuebler/HOPS/Resources)).\\n\\nNecessary when `--metagenomics_profiling_tool malt` and `--metagenomics_run_postprocessing` specified.\n\n Modifies tool parameter(s):\n> - MaltExtract: `-r`",
+                    "fa_icon": "fab fa-buffer"
+                },
+                "metagenomics_maltextract_filter": {
+                    "type": "string",
+                    "default": "def_anc",
+                    "description": "Specify which MaltExtract filter to use.",
+                    "help_text": "Specify which MaltExtract filter to use. This is used to specify what types of characteristics to scan for. The default will output statistics on all alignments, and then a second set with just reads with one C to T mismatch in the first 5 bases. Further details on other parameters can be seen in the [HOPS documentation](https://github.com/rhuebler/HOPS/#maltextract-parameters).\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies tool parameter(s):\n> - MaltExtract: `-f`",
                     "fa_icon": "fas fa-filter",
-                    "description": "Specify the minimum dust score for PRINTSEQ++ complexity filtering.",
-                    "help_text": "Specify the minimum dust score below which low-complexity reads will be removed. A dust score is based on how often different tri-nucleotides occur along a read.\n\n> Modifies PRINSEQ++ parameter: `--lc_dust`"
+                    "enum": ["def_anc", "default", "ancient", "scan", "crawl", "srna"]
+                },
+                "metagenomics_maltextract_toppercent": {
+                    "type": "number",
+                    "default": 0.01,
+                    "description": "Specify percent of top alignments to use.",
+                    "help_text": "Specify frequency of top alignments for each read to be considered for each node.\\nDefault is 0.01, i.e. 1% of all reads (where 1 would correspond to 100%).\\n\\n> :warning: this parameter follows the same concept as `--malt_top_percent` but\\n> uses a different notation i.e. integer (MALT) versus float (MALTExtract)\\n\\nDefault: `0.01`.\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies tool parameter(s):\n> - MaltExtract: `-a`",
```

```suggestion
                    "help_text": "Specify frequency of top alignments for each read to be considered for each node.\\nDefault is 0.01, i.e. 1% of all reads (where 1 would correspond to 100%).\\n\\n> :warning: this parameter follows the same concept as `--malt_top_percent` but\\n> uses a different notation i.e. integer (MALT) versus float (MALTExtract)\\n\\n\\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies tool parameter(s):\n> - MaltExtract: `-a`",
```

Don't need to mention default in the text, it's already in the `default` JSON field